x-pack/filebeat/module/aws/cloudtrail/config/s3.yml

```diff
@@ -2,16 +2,16 @@ type: s3
 queue_url: {{ .queue_url }}
 file_selectors:
 {{ if .process_cloudtrail_logs }}
-  - regex: '^AWSLogs/\d+/CloudTrail/'
```

Comment:
Would removing the `^` anchor fix it? That seems like a simplified form of the fixed version.

Reply:
I was playing in https://regex101.com/r/7WfOYY/1.

Reply:
nice.  thanks.